### PR TITLE
Refactor settings

### PR DESCRIPTION
**Pull request by naveed-ahmad**
* Load setting from cookies 
* Moved all setting related components into setting namespace

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32846

File: src/components/Settings/SwitchToggle/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
```
Comment (naveed-ahmad) [inline src/components/Settings/SwitchToggle/index.js]: we should move this to common components

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32852

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32853

File: src/components/GlobalNav/Surah/index.js [hunk 2]
```diff
@@ -129,38 +96,7 @@ class GlobalNavSurah extends Component {
...
-                  id="setting.fontSize"
-                  defaultMessage="Font Size"
-                />
-              </div>
-              <FontSizeDropdown
-                fontSize={options.fontSize}
-                onOptionChange={setOption}
-              />
-            </Menu>
+            <Settings chapter={chapter} versesIds={versesIds} />
```
Comment (sabeurthabti) [inline src/components/GlobalNav/Surah/index.js]: very nice

File: src/components/Settings/SwitchToggle/index.js (hunk 1 above)
Comment (sabeurthabti) [inline src/components/Settings/SwitchToggle/index.js]: inshallah I will have a look at doing this.

Comment (sabeurthabti) [inline src/components/Settings/SwitchToggle/index.js]: @naveed-ahmad check https://github.com/quran/common-components/tree/master/src/Toggle

update to `quran-components@0.0.75`

Comment (naveed-ahmad) [inline src/components/Settings/SwitchToggle/index.js]: JazakAllah for review and moving this to component @sabeurthabti

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32858

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32859

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32860

**Pull request merged**